routes/calm_outbound.yaml

```diff
@@ -0,0 +1,88 @@
+data:
+  - attributes:
+      created_at: '2021-12-09T15:39:59'
+      destination_override_endpoint: '*'
+      entries:
+        - classifiers: {}
+          config:
+            condition: AND
+            rules:
+              - expression:
+                  field: PathInfo
+                  operator: matches
+                  type: string
+                  values:
+                    - /cards
+          id: 1e61ffde-f985-4ed0-a3ee-200782d81ee9
+          id_selector: null
+          operation: ENRICH
+          operations: null
+          phase: REQUEST
+          public_token_generator: ALPHANUMERIC_SIX_T_FOUR
+          targets:
+            - body
+          token_manager: PERSISTENT
+          transformer: JSON_PATH
+          transformer_config:
+            - $.number
+          transformer_config_map: null
+        - classifiers: {}
+          config:
+            condition: AND
+            rules:
+              - expression:
+                  field: PathInfo
+                  operator: begins_with
+                  type: string
+                  values:
+                    - /cards
+          id: bab64a3d-9fa6-4002-829c-5a4fe7830983
+          id_selector: null
+          operation: REDACT
+          operations: null
+          phase: REQUEST
+          public_token_generator: UUID
+          targets:
+            - body
+          token_manager: PERSISTENT
+          transformer: JSON_PATH
+          transformer_config:
+            - $.account_number
+          transformer_config_map: null
+        - classifiers: {}
```

Comment:
Why do we need this filter?
![image](https://user-images.githubusercontent.com/3820097/145631168-e9ffe074-a347-4378-9d80-ab7c170a20fa.png)
I thought we're not receiving raw data if we aliased it previously

Reply:
We believe we always get raw data in response from CALM.

Reply:
hm... indeed. what does status code not equal 1234 means here?